homeassistant/components/mqtt/cover.py

```diff
@@ -281,14 +281,23 @@ def tilt_message_received(msg):
                 )
 
             if payload.isnumeric() and (
```

Comment:
Or we use separate warnings:
```suggestion
            if not payload.isnumeric():
               _LOGGER.warning("Payload '%s' is not numeric", payload)
               return
            else ...
```

Reply:
Thanks for the feedback! I agree the warning needs clarification, I think the separate warnings gives the best user experience, I'll go with that approach.